src/install/postgresql/linux/install-linux.mdx

```diff
@@ -0,0 +1,51 @@
+---
+componentType: default
+headingText: Configure the PostgreSQL integration
+---
+
+1. Change directory to the integrations configuration folder by running:
+
+  ```shell
+  cd /etc/newrelic-infra/integrations.d
+  ```
+
+2. Copy the sample configuration file by running:
+
+  ```shell
+  sudo cp postgresql-config.yml.sample postgresql-config.yml
+  ```
+
+3. Edit the `postgresql-config.yml` file. This example config file collects all metrics:
+
+  ```yml
+  integrations:
+    - name: nri-postgresql
+      env:
+        USERNAME: postgres
+        PASSWORD: pass
+        HOSTNAME: psql-sample.localnet
+        PORT: 6432
+        DATABASE: postgres
+
+        COLLECT_DB_LOCK_METRICS: false
+        COLLECTION_LIST: 'ALL'
+        TIMEOUT:  10
+      interval: 15s
+      labels:
+        env: production
+        role: postgresql
+      inventory_source: config/postgresql
+  ```
+
+4. Before you restart the infrastructure agent, [create a user](#create-user) with `READ` permissions on the required functions.
```

Comment:
Should this be included in the Yum/Apt/Zypper files too?

Reply:
I've just the question to the help-core-integrations channel.